### ISSUE TITLE
Phase angle bound constraint bug

### ISSUE THREAD

**Issue by hhijazi** [metab0t/opf_benchmark#1]
In `solve_opf` in `poi.py`, there seems to be a bug in the phase angle bound constraints, even though `poi.In` is used, it seems to only be adding half the constraints.
The following diff is a current workaround:
``` diff
-        model.add_linear_constraint(va[f_bus] - va[t_bus], poi.In, angmin[i], angmax[i])
+        model.add_linear_constraint(va[f_bus] - va[t_bus], poi.Leq, angmax[i])
+        model.add_linear_constraint(va[t_bus] - va[f_bus], poi.Leq, -angmin[i])
```

**Comment by metab0t** [comment]
Does this constraint get violated in some cases? I checked the implementation but it seems that I have set the both lower and upper bound of constraints.

**Comment by hhijazi** [comment]
You are right,  I misread the Ipopt log, you are adding both sides of the constraint, Ipopt just counts both inequalities as 1 constraint with lower and upper bounds.
So the only improvement I suggest here is to avoid adding constraints for parallel branches.
